src/config/templates.toml

```
@@ -31,8 +60,9 @@ label(if(current_working_copy, "working_copy"),
   ),
 )
 '''
+builtin_log_comfortable = 'builtin_log_compact + "\n"'
```

```suggestion
builtin_log_comfortable = 'builtin_log_compact ++ "\n"'
```

Otherwise this cannot be expanded.